.github/workflows/percy-workflow.yml

```diff
@@ -7,21 +7,32 @@ on:
     branches:
       - main
     paths:
+      - '**.scss'
+      - 'package.json'
+      - 'src/component-library/**'
+      - 'src/fonts/**'
+      - 'src/build/**.js'
       - 'src/**.js'
+      - '!src/site/_collections/**.js'
+      - '!src/site/_data/**.js'
+      - '!src/site/content/**.js'
       - 'shared/**.js'
-      - '!src/site/_data/countries.js'
-      - 'package.json'
-      - '**.njk'
-      - '**.scss'
+      - 'src/site/content/en/handbook/content-types/**'
+
   pull_request:
     paths:
       - '.github/workflows/percy-workflow.yml'
+      - '**.scss'
+      - 'package.json'
+      - 'src/component-library/**'
+      - 'src/fonts/**'
+      - 'src/build/**.js'
       - 'src/**.js'
+      - '!src/site/_collections/**.js'
+      - '!src/site/_data/**.js'
+      - '!src/site/content/**.js'
       - 'shared/**.js'
-      - '!src/site/_data/countries.js'
-      - 'package.json'
-      - '**.njk'
-      - '**.scss'
+      - 'src/site/content/en/handbook/content-types/**'
```

Comment:
Can you try using a YAML reference for the two `paths` keys? They seem to be the same and would make things easier to see and maintain.